docs/cicero-api.md

```diff
@@ -374,15 +374,15 @@ Returns the package.json for this template.
 #### metadata.getName() ⇒ <code>string</code>
 Returns the name for this template.
 
-**Kind**: instance method of [<code>Metadata</code>](#module_cicero-core.Metadata)  
```

Comment:
Is this generated from script? No need for that change unless it is.

Reply:
Yes, I was trying to revert the changes because it looks like the deployment auto-generates this file.

Reply:
`git reset docs/cicero-api.md` followed by `git checkout docs/cicero-api.md` would do the trick? maybe not worth the effort now that it's already in a commit.